_posts/en/newsletters/2023-01-04-newsletter.md

```diff
@@ -0,0 +1,200 @@
+---
+title: 'Bitcoin Optech Newsletter #232'
+permalink: /en/newsletters/2023/01/04/
+name: 2023-01-04-newsletter
+slug: 2023-01-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Software forks of Bitcoin Core:** last month saw the release of two
+  patchsets on top of Bitcoin Core:
+
+    - *Bitcoin Inquisition:* Anthony Towns [announced][towns bci] to the
+      Bitcoin-Dev mailing list a version of [Bitcoin Inquisition][], a
+      software fork of Bitcoin Core designed to be used on the default
+      [signet][topic signet] for testing proposed soft forks and other
+      significant protocol changes.  This version contains support for
+      the [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] and
+      [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify] proposals.
+      Towns's email also includes additional information that will be
+      useful to anyone participating in the signet tests.
+
+    - *Full-RBF peering node:* Peter Todd [announced][todd rbf node] a
+      patch on top of Bitcoin Core 24.0.1 that sets a full-RBF service
```

Comment:
Could link here as it could be interesting for readers to see the other service bits 

https://github.com/petertodd/bitcoin/commit/c15b8d70778238abfa751e4216a97140be6369af